### PR TITLE
[Updated] NodeBalancer Overview

### PR DESCRIPTION
**Pull request by wildmanonline**
Updated the NodeBalancer Overview page

### CODE REVIEW

File: docs/guides/platform/nodebalancer/what-are-nodebalancers/index.md [hunk 1]
```diff
@@ -47,12 +47,4 @@ The optimal solution for a highly available site or application is to have multi
...
-- Configuration be added to ensure the same client requests are routed to the same backend Linode, this is called [sticky sessions](/docs/platform/nodebalancer/nodebalancer-reference-guide/#session-stickiness).
-
-## Limitations
-
-- Nodebalancers have a maximum connection limit of 10,000 concurrent connections.
-
-## Next Steps
-
-For more on NodeBalancers see the [Getting Started with NodeBalancers](/docs/platform/nodebalancer/getting-started-with-nodebalancers/) guide and the [NodeBalancer Reference](/docs/platform/nodebalancer/nodebalancer-reference-guide/) guide.
+- Configuration be added to ensure the same client requests are routed to the same backend Linode, this is called [sticky sessions](/docs/platform/nodebalancer/nodebalancer-reference-guide/#session-stickiness).
```
Comment (andystevensname) [inline docs/guides/platform/nodebalancer/what-are-nodebalancers/index.md:50]: Wonky opening to sentence

File: docs/products/networking/nodebalancers/_index.md [hunk 2]
```diff
@@ -1,40 +1,81 @@
...
 
-NodeBalancers are available across [all regions](https://www.linode.com/global-infrastructure/).
+## High Availability
 
-## Pricing
+In a typical single machine configuration, issues with the machine may cause the application to stop working as expected or become inaccessible. High availability solutions remove this single point of failure through combining multiple machines (redundancy), monitoring systems, and automatic failover - all of which are implemented by NodeBalancers.
+
+## Horizontal Scaling
+
+There are two main ways to scale an application, increasing the performance and capacity to your applications. *Vertical scaling* increases or decreases the resources on the existing machines. This is achieved by [resizing](/docs/guides/resizing-a-linode/) your Compute Instances. *Horizontal scaling* adds or remove machines that are identically configured to serve your application or perform a certain task. This is commonly accomplished through a load balancing solution, like NodeBalancers. Horizontal scaling can be much more flexible and allows you to scale as needed without taking down your site while upgrading or downgrading.
```
Comment (andystevensname) [inline docs/products/networking/nodebalancers/_index.md:21]: I think in the opening sentence, you can remove the comma and say "to increase." As it stands, it sounds like the first sentence is giving one of the ways to scale, which it does not.

**Review by andystevensname — CHANGES_REQUESTED** [review]

File: docs/products/networking/nodebalancers/_index.md (hunk 2 above)
Comment (wildmanonline) [reply docs/products/networking/nodebalancers/_index.md:21]: Implemented

File: docs/guides/platform/nodebalancer/what-are-nodebalancers/index.md (hunk 1 above)
Comment (wildmanonline) [reply docs/guides/platform/nodebalancer/what-are-nodebalancers/index.md]: Updated to

> Client requests can be routed to the same backend Linode through [sticky sessions](/docs/platform/nodebalancer/nodebalancer-reference-guide/#session-stickiness).

**Review by andystevensname — APPROVED** [review]

**Review by bbiggerr — APPROVED** [review]

**Pull request merged**